src/censys/cloud_connectors/aws_connector/connector.py

```diff
@@ -88,89 +103,164 @@ def __init__(self, settings: Settings):
         self.cloud_asset_scanners = {
             AwsResourceTypes.STORAGE_BUCKET: self.get_s3_instances,
         }
-
-        self.ignored_tags = []
         self.global_ignored_tags: set[str] = set(IGNORED_TAGS)
 
-    def scan_seeds(self):
-        """Scan AWS for seeds."""
+    def scan_seeds(self, **kwargs):
+        """Scan AWS."""
+        scan_context: AwsScanContext = kwargs["scan_context"]
+
+        # multiprocessing requires separate logger instances per process
+        # TODO: there is still something odd going on here, notice logger isnt being set to self.logger, but by even calling get_logger it "fixes" the sub-process log level from being WARNING to .env's DEBUG
+        logger = get_logger(
+            log_name=f"{self.provider.lower()}_cloud_connector",
+            level=self.settings.logging_level,
+            provider=f"{self.provider}_{scan_context.account_number}_{scan_context.region}",
+        )
+        scan_context.logger = logger
+
         self.logger.info(
-            f"Scanning AWS account {self.account_number} in region {self.region}"
+            f"Scanning AWS - account:{scan_context.account_number} region:{scan_context.region}"
```

Comment:
maybe specify "scanning for seeds" here? otherwise we'll have the same log twice and not know which was for seeds vs cloud assets. not sure if that's a big deal